### PR TITLE
#574: Tooltips for fields

### PR DESCRIPTION
**Pull request by suvarnakale**

### CODE REVIEW

File: src/app/forms/types/tooltip.type.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        <small class="form-text text-muted">{{ to.description }}</small>
+        
+        
+    </div>  
+`
+})
+export class TooltipWrapper extends FieldType {
+  tooltipWrapper: any;
+  tooltipText: any;
+  get tooltipContent(): string {
```
Comment (Pratikshakhandagale) [inline src/app/forms/types/tooltip.type.ts:63]: @suvarnakale is it requred?

File: src/app/forms/types/tooltip.type.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        
+        <span class="tooltip-text">{{ to.tooltip }}</span>
+        <input type="text" class="form-control" [formControl]="formControl" [formlyAttributes]="field" [title]="to.tooltip">
+        <small class="form-text text-muted">{{ to.description }}</small>
+        
+        
+    </div>  
+`
+})
+export class TooltipWrapper extends FieldType {
```
Comment (Pratikshakhandagale) [inline src/app/forms/types/tooltip.type.ts:60]: @suvarnakale - is this tooltip work for fiedset and field too?

File: src/app/forms/types/tooltip.type.ts (hunk 1 above)
Comment (suvarnakale) [reply src/app/forms/types/tooltip.type.ts:63]: I added that for mouse hover and mouseOut but it is also managable by css, added later. I will remove it.

File: src/app/forms/types/tooltip.type.ts (hunk 2 above)
Comment (suvarnakale) [reply src/app/forms/types/tooltip.type.ts:60]: This works for field only

Comment (Pratikshakhandagale) [reply src/app/forms/types/tooltip.type.ts:59]: So can you change the class name with **TooltipType**

File: src/app/forms/types/tooltip.type.ts (hunk 1 above)
Comment (Pratikshakhandagale) [reply src/app/forms/types/tooltip.type.ts:60]: so are you using this rmethod right now or not?

Comment (suvarnakale) [reply src/app/forms/types/tooltip.type.ts:60]: Not using the get method.. will remove it.. previously I was confused with varible declarations and function.

**Review by Pratikshakhandagale — APPROVED** [review]

**Pull request merged**